src/Simulator/IncrementalRedstoneSimulator/PressurePlateHandler.h

```diff
@@ -36,6 +37,11 @@ class cPressurePlateHandler : public cRedstoneHandler
 					FoundPlayer = true;
 				}
 
+				if (a_Entity.IsPickup())
+				{
+					NumberOfEntities += dynamic_cast<cPickup*>(&a_Entity)->GetItem().m_ItemCount;
```

Comment:
Make `NumberOfEntities` an `int` to fix the clang warning.

Reply:
I don't get a warning there but why not. :)

Reply:
Ah you mean the check complained, I just saw it now.